python/ray/_raylet.pyx

```diff
@@ -1198,14 +1205,17 @@ cdef class CoreWorker:
                     Buffer.make(data))
             if self.is_local_mode or (put_small_object_in_memory_store
                and <int64_t>total_bytes < put_threshold):
+                if not self.is_local_mode:
```

Comment:
i might miss the context of local mode. why local mode doesn't need contained_object_refs?

Reply:
Ah I was trying to fix a CI error that I don't think is actually related.

I don't really know how local mode works, but I don't think that it uses plasma at all, so it probably doesn't matter if we include contained_object_refs or not. Anyway, I'll just remove this.